app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -41,9 +41,9 @@ public void onClick(View view) {
                     Toast.makeText(view.getContext(), "What did you click?? :P", Toast.LENGTH_SHORT).show();
                 }
 
-                if (url.equals(null)) {
+                if (url.equals(null) || url.isEmpty()) {
```

Comment:
@mananwason We do need check if `url` is a empty string (i.e "") even if it's not `null`.